web/client/components/data/query/CrossLayerFilter.jsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2017, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+
+const Message = require('../../I18N/Message');
+
+const SwitchPanel = require('../../misc/switch/SwitchPanel');
+const {Row, Col} = require('react-bootstrap');
+const Select = require('react-select');
+
+
+const GeometricOperationSelector = require('./GeometricOperationSelector');
+const GroupField = require('./GroupField');
+
+
+const isSameOGCServiceRoot = (origSearchUrl, {search, url} = {}) => origSearchUrl === url || origSearchUrl === (search && search.url);
+
+// bbox make not sense with cross layer filter
+const getAllowedSpatialOperations = (spatialOperations) => (spatialOperations || []).filter( ({id} = {}) => id !== "BBOX");
```

Comment:
This excludes the BBOX because makes not sense in the crossLayer context. We could improve an override in the future for the spatialOperations passed to the nested component, if someone need this.